modules/graphstudio/pages/design-schema.adoc

```diff
@@ -85,64 +86,66 @@ when accessing these attributes.
-types. First, click the source vertex type. A hint will appear on the
-vertex type circle:
+types.
+First, click the source vertex type.
+A hint will appear on the vertex type circle:
 
 image:click_edge_type_source_vertex.png[image]
 
-Then click the target vertex type. The add edge type panel will appear to the right:
+Click the target vertex type. The add edge type panel will appear to the right:
 
 image:add-edge-panel.png[image]
 
-You must specify an edge type name. The source vertex type and target
-vertex type are selected based on your clicking action. However, you can
-change that by choosing another vertex type in the dropdown list.
-
-You can also click the green plus sign at the right of the source and
-target vertex types section to add more source and target vertex types
-of the edge type.
-
-image:edge-multi-pair.png[image]
-
-By default, the edge type is undirected. To make the edge type directed,
-mark the Directed checkbox:
+The edge type name cannot be left blank.
 
-image:directed-edge.png[image]
-
-If Directed is checked, another checkbox will appear for you to choose
-whether the edge type should include reverse edges.
+By default, the edge type is undirected.
+To make the edge type directed, mark the Directed checkbox, which will bring up another checkbox will appear for you to choose whether the edge type should include reverse edges.
 Including reverse edges provides more flexibility when designing queries.
 
-Unselect the reverse edge checkbox ONLY IF your machine memory is very tight, because if there is no reverse edge, queries will not be able to traverse
+The source and target vertex types are selected based on the vertices you clicked, but you can change that by choosing another vertex type in the dropdown list.
+Click the green plus sign at the right of the source and
+target vertex types section to add more source and target vertex types.
+
+Unselect the reverse edge checkbox ONLY IF your machine memory is very limited, because if there is no reverse edge, queries will not be able to traverse
 backwards along this directed edge type from the target vertex to the
 source vertex.
 
-Editing edge type attributes is the same as editing vertex type
-attributes.
-
 Once you are satisfied with the edge type settings, click the checkmark button at the top to add the edge
 type, which will instantly appear in the working panel.
 
-You can add multiple edge types between the same source vertex type and
-target vertex type pair. Moreover, an edge can use the same vertex type
-for both its source vertex type and its target vertex type, e.g., a
-Friendship edge from a Person vertex to Person vertex.
+=== Edge attributes
+
+image::edge-attributes.png[]
+
+Add an attribute to your edge in the Attributes panel.
+Edge attributes work just like vertex attributes.
+
+The btn:[Discriminator] checkbox allows you to enable multiple edges with the same attribute type to exist between the same vertex pair.
```

Comment:
The combination of  discriminators act like an edge id. And it allows multiple edges of the same type(not attribute type) to exist between any two vertex pair, if the discriminator attribute values are unique.